packages/dispatcher/src/index.ts

```diff
@@ -12,12 +14,15 @@ import logger from "./logger";
 
 export class SlackDispatcher {
   private app: App;
-  private jobManager: KubernetesJobManager;
+  private jobManager?: KubernetesJobManager;
+  private queueProducer?: QueueProducer;
   private repoManager: GitHubRepositoryManager;
   private config: DispatcherConfig;
+  private isQueueMode: boolean;
```

Comment:
get rid of this isQueue mode, we only support Queue mode nothing else to keep it simple